computation/src/main/java/com/powsybl/computation/ComputationParametersImpl.java

```diff
@@ -6,23 +6,25 @@
  */
 package com.powsybl.computation;
 
-import java.util.*;
+import com.powsybl.commons.extensions.AbstractExtendable;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+import java.util.OptionalLong;
 
 /**
  * @author Yichen TANG <yichen.tang at rte-france.com>
  */
-public class ComputationParametersImpl implements ComputationParameters {
+public class ComputationParametersImpl extends AbstractExtendable<ComputationParameters> implements ComputationParameters {
 
     private final Map<String, Long> timeoutMap;
 
     private final Map<String, Long> deadlineMap;
 
-    private final Map<String, String> qosMap;
-
-    ComputationParametersImpl(Map<String, Long> timeoutMap, Map<String, Long> deadlineMap, Map<String, String> qosMap) {
+    ComputationParametersImpl(Map<String, Long> timeoutMap, Map<String, Long> deadlineMap) {
```

Comment:
A not important detail again, but maybe we should discuss this at powsybl level: should we have guidelines for naming ? @mathbagu ? 

In my opinion, it's not worth it to mention the type in the name of variables (timeoutMAP), same thing for the fields.
Maybe timeouts is enough (or timeoutsForCommand to explain what's the key part of the map).

Reply:
Agree: it’s a bad idea to suffix the name of the variable by its type. If the type change you will forget to fix the name of the variable.